test/IniModule.test.cpp

```diff
@@ -0,0 +1,53 @@
+#include "FileSystemHelper.h"
+#include "FsInclude.h"
+#include "IniFile.h"
+#include "ModuleLoader.h"
+#include <fstream>
+#include <string>
+#include <string_view>
+#include <gtest/gtest.h>
+
+
+// sectionPairs is a non-line terminated list of keys and values. EG: TestModule = No
+void WriteExternalModuleIniFile(std::string_view iniFilename, const std::vector<std::string_view> sectionPairs)
+{
+	std::ofstream iniFileStream(iniFilename);
+
+	iniFileStream << "[ExternalModules]\n";
+
+	for (const auto& sectionPair : sectionPairs)
+	{
+		iniFileStream << sectionPair << "\n";
+		//iniFileStream << "Test = yes\n";
+	}
+
+	iniFileStream.close();
+}
+
+TEST(IniModule, NoDll)
+{
+	const auto iniFilename = fs::path(GetExeDirectory()) / fs::path("IniModuleTest.ini");
+	WriteExternalModuleIniFile(iniFilename.string(), { "Test = yes" });
```

Comment:
I'm not sure it is legal to pass into a string_view argument into a function without first recording the string to variable. Seems to work fine on MSVC though.

Reply:
Looks like the standard library packaged with Mingw doesn't have an overload taking `std::string_view` for the `std::ofstream` constructor. I guess we can just use a `const std::string&` for now.

Reply:
Okay, the other option would be to define our own overload of ofstream for mingw that takes a string_view. :-) I know how to do that in C#, not sure what the C++ equivalent is without researching (I assume it is possible).

Thanks for fixing for me.

Reply:
We would have to add/modify things in the `std::` namespace, which creates upgrade and portability issues. Probably best to just leave this one alone.

It's also likely we'd need to do a `std::string_view` to `std::string` conversion to get it working, which kind of defeats the purpose of using `std::string_view` in the first place.